tests/test_util.py

```diff
@@ -13,3 +14,14 @@ def test_get_latest_version_timeout(mock_pypi_request_timeout):
     start = time()
     assert get_latest_version() is None
     assert (time() - start) < 0.5
+
+
+@pytest.mark.usefixtures("enable_docker_pull")
+def test_image_is_latest(docker_client):
+    """Test that the latest version is identified correctly."""
+    # download the alpine image for testing
+    image_name = "alpine:latest"
```

Comment:
Let's use something more lightweight, perhaps the hello-world image from Docker?

Also, it is not clear to me, is this image then automatically deleted in `enable_docker_pull` fixture?

Reply:
>  is this image then automatically deleted in enable_docker_pull fixture?

No, I don't think so. This fixture is to guarantee the test will be skipped if it contains docker pull.

Reply:
What a surprise, the `hello-world` is bigger than `alpine`.  :smile: 
- https://hub.docker.com/_/hello-world/tags
- https://hub.docker.com/_/alpine/tags